server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -69,4 +80,16 @@ Authenticator provideAuthenticator(MetricFactory metricFactory,
             basicAuthenticationStrategy,
             jwtAuthenticationStrategy);
     }
+
+    @Provides
+    @Singleton
+    JmapRfc8621Configuration provideConfiguration(PropertiesProvider propertiesProvider) throws ConfigurationException {
+        try {
+            Configuration configuration = propertiesProvider.getConfiguration("jmap");
+            return JmapRfc8621Configuration.from(configuration);
+        } catch (FileNotFoundException e) {
+            LOGGER.warn("Could not find JMAP configuration file [jmap.properties]. JMAP server will be enabled with default value.");
```

Comment:
This log message is still inappropriate.

Reply:
which message you suggest?

Reply:
There is already another log for missing jmap.properties hence we don't need to log here too.